### DISCUSSION TITLE
apt-key is now deprecated

### DISCUSSION THREAD

**Discussion by Fuseteam** [mcdope/pam_usb#246] [General]
the instructions on https://apt.mcdope.org/ for importing the packaging key is outdated on newer debian/ubuntu
 
we get this warning now when we run `sudo apt-key adv --recv-keys --keyserver keyserver.ubuntu.com 913558C8A5E552A7`: `Warning: apt-key is deprecated. Manage keyring files in trusted.gpg.d instead (see apt-key(8)).`

this post explains the reasoning: https://askubuntu.com/a/1307181/414411

TLDR; the new instructions are:
- `sudo gpg --no-default-keyring --keyring /usr/share/keyrings/mcdope-achive-keyring.gpg --keyserver keyserver.ubuntu.com --recv-keys 913558C8A5E552A7` to import the key
- the sources.list entry is now `deb [signed-by=/usr/share/keyrings/mcdope-achive-keyring.gpg] https://apt.mcdope.org/ ./`

**Comment by mcdope** [comment]
Nice! Thanks.

**Comment by mcdope** [comment]
Closing this, since the "issue" (#247) caused by this is resolved. 

Thanks again for providing an insta-fix instead of just reporting it :partying_face: I already noticed it some months ago, but was just to lazy to care about a deprecation :smiling_face_with_tear: 

If you're ever in Dortmund, Germany make sure to message me - we gonna have some drinks on me for all your help ^^

**Reply by Fuseteam** [reply]
haha thanks for the quick fix, happy i could be of help :D
